src/model.coffee

```diff
@@ -51,6 +56,13 @@ class window.Brainstem.Model extends Backbone.Model
 
 
   #
+  # Init
+
+  constructor: (options = {}) ->
+    super
+    @storageManager = StorageManager.get()
```

Comment:
The StorageManager is a singleton. I want too keep everything self contained in this package. The refactor will involve completely moving loading logic from the storage manager into the collection and model and their respective loaders.